stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js

```diff
@@ -0,0 +1,129 @@
+const SPACINGS = [
+  "--spacing-xs",
+  "--spacing-small",
+  "--spacing-medium",
+  "--spacing-large",
+  "--spacing-xl",
+  "--spacing-xxl",
+  "--spacing-xxxl",
+];
+
+const BOX_SHADOWS = ["--box-shadow-small", "--box-shadow-medium", "--box-shadow-large"];
+
+const BORDER_RADIUSES = ["--border-radius-small", "--border-radius-medium", "--border-radius-big"];
```

Comment:
We do use border-radius 50% in some cases (avatar and more components).
it is not part of our system. also in use within the monolith (354 times).
so this is a false positive.

Reply:
I'm not sure I understand - why is this false positive? there shouldn't be an error if a value without a matching var is used. If there WAS a var with had the value of `50%`, only then we would have shown an error (and it was "true-positive" 😄  )

Reply:
ok. but we do want to enforce use of vibe values dont we?
lets talk about this briefly in person.